File: sdk/rust/examples/online_get.rs
use oomclient::{Client, OnlineGetFeatures::*};

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let mut client = Client::connect("http://localhost:50051").await?;

    let features = FeatureNames(vec!["account.state".into(), "txn_stats.count_7d".into()]);
    let response = client.online_get_raw("48", features.clone()).await?;
    println!("RESPONSE={:#?}", response);

    let response = client.online_get("48", features).await?;
    println!("RESPONSE={:#?}", response);

    Ok(())
}

File: sdk/python_v2/examples/online_get.py
#!/usr/bin/env python3
import asyncio
from oomclient import Client

async def main():
    client = await Client.connect("http://localhost:50051")
    features = ["account.state", "transaction_stats.transaction_count_7d"]
    result = await client.online_get("48", features)
    print(result)

asyncio.run(main())
